src/components/ButtonGroup/index.js

```diff
@@ -0,0 +1,191 @@
+import React, { Fragment } from 'react';
+import CollapsedGroup from './collapsedGroup';
+
+import './scss/ButtonGroup.scss';
+
+type PropsType = {
+  alt?: boolean,
+  children?: any,
+  openOnFirstChild?: boolean,
+  numButtons?: number,
+  disableButtons?: boolean,
+  toggleButtonContents?: React$Element,
+};
+
+type DefaultPropsType = {
+  children: any,
+  openOnFirstChild: boolean,
+  alt: false,
+  numButtons: number,
+  disableButtons: boolean,
+};
+
+type StateShape = {
+  expand: boolean,
+};
+
+class ButtonGroup extends React.Component<PropsType, StateShape> {
+  static defaultProps = {
+    baseClass: 'button-group',
+    children: [],
+    openOnFirstChild: false,
+    alt: false,
+    numButtons: 2,
+    disableButtons: false,
+    toggleButtonContents: <span>Menu</span>,
+  };
+
+  defaultProps: DefaultPropsType;
+
+  constructor(props: PropsType) {
+    super(props);
+    this.state = {
+      expand: false,
+    };
+  }
+
+  // Return non undefined/null props.children
+  getChildrenAsArray = (start = 0, end = 0) =>
+    React.Children.toArray(this.props.children)
+      .filter(child => child !== undefined || child !== null)
```

Comment:
this could actually be simplified as just `true` :)
`.filter(child => child !== undefined || true)`

Reply:
I don't think it can-

Given `[undefined, null, null, 'hi']`

`.filter(child => child !== undefined || true)` will return-

`[ undefined, null, null, "hi" ]`